docs/provider-pkcs11.7

```diff
@@ -170,17 +170,26 @@ immediately at application startup)
 .PP
 Workarounds that may be needed to deal with some tokens and cannot be
 autodetcted yet are not appropriate defaults.
+.SS no-deinit
 .PP
-The only quirk currently implemented is \[lq]no-deinit\[rq] It prevent
-de-initing when OpenSSL winds down the provider.
+It prevents de-initing when OpenSSL winds down the provider.
 NOTE this option may leak memory and may cause some modules to misbehave
 if the application intentionally unloads and reloads them.
+.SS no-operation-state
+.PP
+OpenSSL by default assumes contexts with operations in flight can be
+easily duplicated.
+That is only possible is the tokens support getting and setting the
+operation state.
+If the quirk is enabled the context duplication only performs a context
+move.
```

Comment:
Just say "Context Duplication is not performed" because of: #209 which I plan to merge sometime soon once I verify my patch to deal with this properly is actually in openssl 3.2.0